### PR TITLE
update vrf classes

### PR DESCRIPTION
**Pull request by jli113**
Adding the following entities:
https://brickschema.org/schema/Brick#Hydrocarbon_Refrigerant,"Hydrocarbon Refrigerants are natural, nontoxic refrigerants that have no ozone depleting properties and absolutely minimal global warming potential. The most efficient and environmentally safe refrigerants in the world are the five natural refrigerants which are Air, Water, Carbon Dioxide, Ammonia and Hydrocarbons.",https://www.engas.com.au/about/about-hydrocarbon-refrigerants/
https://brickschema.org/schema/Brick#Fluorocarbon_Refrigerant,"Although they can be found in nature, the gases used are industrial gases, just like HFCs & HFOs, and are produced in refineries or other industrial facilities.",
https://brickschema.org/schema/Brick#CFC,"Chlorofluorocarbons (CFCs): when derived from methane and ethane these compounds have the formulae CClmF4−m and C2ClmF6−m, where m is nonzero. More than 98% of the CFCs used in MDIs have already been phased out, and will disappear completely in 2016, a significant milestone following 30 years of concerted global action to protect the ozone layer.",
https://brickschema.org/schema/Brick#HCFC,"Hydro-chlorofluorocarbons (HCFCs): when derived from methane and ethane these compounds have the formula CClmFnH4−m−n and C2ClxFyH6−x−y, where m, n, x, and y are nonzero. New production and import of most HCFCs were phased out as of 2020. The most common HCFC in use today is HCFC-22 or R-22, a refrigerant still used in existing air conditioners and refrigeration equipment. ",
https://brickschema.org/schema/Brick#HFC,"Hydrofluorocarbons (HFCs): when derived from methane, ethane, propane, and butane, these compounds have the respective formulae CFmH4−m, C2FmH6−m, C3FmH8−m, and C4FmH10−m, where m is nonzero. An HFC phase-down under the Montreal Protocol was agreed in Kigali, Rwanda, in October 2016. Developed countries will phase-down their production and imports of HFCs by 85% between 2019 and 2036. Developing countries will also phase-down their HFC production and imports.",
https://brickschema.org/schema/Brick#HFO,"Hydrofluoroolefins (HFOs) are unsaturated organic compounds composed of hydrogen, fluorine and carbon.",https://en.wikipedia.org/wiki/Hydrofluoroolefin
https://brickschema.org/schema/Brick#R410A,"R-410A, sold under the trademarked names AZ-20, EcoFluor R410, Forane 410A, Genetron R410A, Puron, and Suva 410A, is a zeotropic but near-azeotropic mixture of difluoromethane (CH2F2, called R-32) and pentafluoroethane (CHF2CF3, called R-125) that is used as a refrigerant in air conditioning applications",https://en.wikipedia.org/wiki/R-410A
https://brickschema.org/schema/Brick#R134a,"1,1,1,2-Tetrafluoroethane (also known as norflurane (INN), R-134a, Freon 134a, Forane 134a, Genetron 134a, Green Gas, Florasol 134a, Suva 134a, or HFC-134a) is a hydrofluorocarbon (HFC) and haloalkane refrigerant with thermodynamic properties similar to R-12 (dichlorodifluoromethane) but with insignificant ozone depletion potential and a lower 100-year global warming potential (1,430, compared to R-12's GWP of 10,900).[2] It has the formula CF3CH2F and a boiling point of −26.3 °C (−15.34 °F) at atmospheric pressure.","https://en.wikipedia.org/wiki/1,1,1,2-Tetrafluoroethane"
https://brickschema.org/schema/Brick#R1234yf,"2,3,3,3-Tetrafluoropropene, HFO-1234yf, is a hydrofluoroolefin (HFO) with the formula CH2=CFCF3. It is also designated R-1234yf as the first of a new class of refrigerants: it is marketed under the name Opteon YF by Chemours and as Solstice YF by Honeywell.","https://en.wikipedia.org/wiki/2,3,3,3-Tetrafluoropropene"
https://brickschema.org/schema/Brick#Liquid_Hydrocarbon_Refrigerant,,
https://brickschema.org/schema/Brick#Liquid_Hydrofluoric_Refrigerant,,
https://brickschema.org/schema/Brick#Liquid_CFC,,
https://brickschema.org/schema/Brick#Liquid_HCFC,,
https://brickschema.org/schema/Brick#Liquid_HFC,,
https://brickschema.org/schema/Brick#Liquid_HFO,,
https://brickschema.org/schema/Brick#Liquid_R410A,,
https://brickschema.org/schema/Brick#Liquid_R134a,,
https://brickschema.org/schema/Brick#Liquid_R1234yf,,
https://brickschema.org/schema/Brick#Refrigerant_System,"Refrigerant systems operate on the direct expansion (DX) principle meaning that heat is transferred to or from the space directly by circulating refrigerant to evaporators located near or within the conditioned space.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#Single_Split_System,"Split type air conditioning systems are one-to-one systems consisting of one evaporator (fan coil) unit connected to an external condensing unit. Both the indoor and outdoor units are connected through copper tubing and electrical cabling.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#Multi_Split_System,"A multi-type air conditioning system operates on the same principles as a split type air-conditioning system with multiple indoor units connected to one external outdoor unit.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#VRF_System,"VRF systems connect the outdoor section to several the indoor sections. VRF systems continually adjust the flow of refrigerant to each indoor evaporator. The control is achieved by continually varying the flow of refrigerant through a pulse modulating valve (PMV) whose opening is determined by the microprocessor receiving information from the thermistor sensors in each indoor unit. The indoor units are linked by a control wire to the outdoor unit which responds to the demand from the indoor units by varying its compressor speed to match the total cooling and/or heating requirements.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#VRF_Heat_Pump_System,"VRF heat pump systems permit heating or cooling in all of the indoor units but NOT simultaneous heating and cooling. When the indoor units are in the cooling mode, they act as evaporators; when they are in the heating mode, they act as condensers. These are also known as two-pipe systems.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#VRF_Heat_Recovery_System,"Variable refrigerant flow systems with heat recovery (VRF-HR) capability can operate simultaneously in heating and/or cooling mode, enabling heat to be used rather than rejected as it would be in traditional heat pump systems.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#Indoor_Unit,Indoor boxes with heat exchangers,
https://brickschema.org/schema/Brick#Outdoor_Unit,Outdoor boxes with heat exchangers and compressors,
https://brickschema.org/schema/Brick#Mode_Switch_Box,"MS boxes with a built-in solenoid valves switch the modes of operation of indoor units, allowing different groups of units connected to the MS box to work simultaneously in different modes: cooling or heating, with precisely selected performance.",https://midea.mt/index.php/product/mode-switch-box/
https://brickschema.org/schema/Brick#hasConnection,"The subject is linked with the object in the context of some sequential process; some media is between them.",
https://brickschema.org/schema/Brick#Four_Way_Valve,"The four-way valve or four-way cock is a fluid control valve whose body has four ports equally spaced round the valve chamber and the plug has two passages to connect adjacent ports.",https://en.wikipedia.org/wiki/Four-way_valve
https://brickschema.org/schema/Brick#Check_Valve,"A check valve, non-return valve, reflux valve, retention valve, foot valve, or one-way valve is a valve that normally allows fluid (liquid or gas) to flow through it in only one direction.",https://en.wikipedia.org/wiki/Check_valve
https://brickschema.org/schema/Brick#Electronic_Expansion_Valve,"The electronic expansion valve (EEV) operates with a much more sophisticated design. EEVs control the flow of refrigerant entering a direct expansion evaporator. They do this in response to signals sent to them by an electronic controller.",https://www.achrnews.com/articles/95056-electronic-expansion-valves-the-basics
https://brickschema.org/schema/Brick#Solenoid_Valve,"A solenoid valve is an electromechanically operated valve. Solenoid valves differ in the characteristics of the electric current they use, the strength of the magnetic field they generate, the mechanism they use to regulate the fluid, and the type and characteristics of fluid they control. The mechanism varies from linear action, plunger-type actuators to pivoted-armature actuators and rocker actuators. The valve can use a two-port design to regulate a flow or use a three or more port design to switch flows between ports. Multiple solenoid valves can be placed together on a manifold.",https://en.wikipedia.org/wiki/Solenoid_valve
https://brickschema.org/schema/Brick#Header,"The refrigerant pipe-work uses a number of separation tubes and/or headers, a header has more than 2 branches.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#Separation_Tube,"The refrigerant pipe-work uses a number of separation tubes and/or headers, a separation tube has 2 branches.",https://www.seedengr.com/Variable%20Refrigerant%20Flow%20Systems.pdf
https://brickschema.org/schema/Brick#Capillary_Tube,"Capillary tube is one of the most commonly used throttling devices in the refrigeration and the air conditioning systems. The capillary tube is a copper tube of very small internal diameter. It is of very long length and it is coiled to several turns so that it would occupy less space. The internal diameter of the capillary tube used for the refrigeration and air conditioning applications varies from 0.5 to 2.28 mm (0.020 to 0.09 inches). Capillary tube used as the throttling device in the domestic refrigerators, deep freezers, water coolers and air conditioners.",https://www.brighthubengineering.com/hvac/58420-capillary-tube-for-refrigeration-and-air-conditioning-systems/
https://brickschema.org/schema/Brick#Drier,"A drier as its name imply is used to remove the moisture from the refrigerant.",https://www.airconditioning-systems.com/filterdriers.html
https://brickschema.org/schema/Brick#Accumulator,"A suction accumulator is used to prevent liquid refrigerant floodback to the compressor. The accumulator protects the system components. It is located on the low-pressure side of the circuit between the evaporator outlet and the compressor suction port.",https://hvacrschool.com/whast-is-a-suction-accumulator/

### CODE REVIEW

**Comment by jli113** [comment]
It occurred to me that there are some packaged units called window/through_wall/self-contained air conditioners, which contain indoor units and outdoor units.

**Review by jbkoh — CHANGES_REQUESTED** [review]
Thanks for the contribution. This would be very useful, though I realized that there are several concepts that the contributor misunderstood from Brick's original intention. I left several comments, but they are not complete. I will leave general comments here and review this again after the modification.

Tags are conceptual component of the class. You can check here: https://docs.brickschema.org/brick/concepts.html. I see patterns where you use tags as a placeholder for possible relations of the class (e.g., subcomponents or points of a target equipment). However, Brick currently doesn't model that, though it's in our future road map. Check that and please update the PR.

Also, in Brick's general design principle, we focus on concepts with actual use cases. E.g., though I understand what `Header` means but its use cases are not clear. Please review the proposed concepts with unclear use cases.

The current PR interchangeably use Systems and Equipment. Please check the documentation and distinguish them. I understand some types of equipment are often considered as a system, but the documentation would help you understand the difference in our modeling.

Finally, please put some examples with the proposed design. We normally don't mandate adding examples, but given that the current proposal is exhaustive, an example would be super helpful.

Thanks again for the huge contribution and I'd love to make this aligned with Brick soon.

**Comment by connorjcantrell** [comment]
The term "Indoor Unit" is equivalent to Brick's existing "Fan Coil Unit".

The term "Outdoor Unit" is also referred to as a "Condensing Unit", which needs to be added to Brick. Condensing Unit is not to be confused with Condenser, which is a core component of a Condensing Unit along with a compressor and sometimes a reversing valve.

We can further distinguish the type of Condensing Unit, such as:
- Heat Pump Condensing Unit
- Cooling Only Condensing Unit 

We can also improve our ability to represent the various types of Indoor Units/ Fan Coil Units by defining the following:
- Wall [Mounted] Fan Coil Unit
Screenshot 2023-10-30 at 11 20 22 AM

- Ceiling [Mounted] Fan Coil Unit
Screenshot 2023-10-30 at 11 17 01 AM
- Floor [Mounted] Fan Coil Unit
Screenshot 2023-10-30 at 11 17 10 AM

- Concealed/ Duct Fan Coil Unit
Screenshot 2023-10-30 at 11 18 53 AM


It is worth mentioning that up to this point, all concepts can be used interchangeably with conventional cooling and heating systems and VRF systems. It only becomes a VRF system when "Branch Selector" are in between the FCUs and Condensing Units. Therefore, Brick will also need to introduce a Branch Selector Box class. 

In summary, I propose introducing:
- Condensing Unit
- Branch Selector
- Wall Fan Coil Unit
- Ceiling Fan Coil Unit
- Floor Fan Coil Unit
- Duct Fan Coil Unit

**Comment by connorjcantrell** [comment]
It's worth mentioning that the previous FCUs are focused on Direct Expansion (DX) coils. It is also important to incorporate water-based systems into our Fan Coil Unit taxonomy, although that topic falls outside the scope of this thread.

**Comment by gtfierro** [comment]
Closing so we can continue development in a single location on a more recent version of Brick (see #589)
